.jshintrc

```diff
@@ -1,42 +1,37 @@
 {
```

Comment:
We use eslint + airbnb style for all JS packages. Shall we just move to eslint + prettier?

Reply:
Prettier is definitely the standard for JS.

ESLint is more for TypeScript and ECMAScript and other JS derivatives. JSHint is for JS.

ESLint _can_ operate on JS, but it has a boatload of dependencies, very complex configuration, etc.

That said, ESLint is better than _nothing_, but it's nowhere near as good as JSHint for JavaScript. JSHint is a single dependency, no plugins, no formatting, no rewrites, no style, etc.

Reply:
I think the most important here is a consistency. All JS repos in this org should follow the same code style. Then we should go with the same configuration for eslint what we in https://github.com/dashpay/platform/ and prettier should follow eslint config https://github.com/prettier/eslint-config-prettier

Reply:
> All JS repos in this org should follow the same code style

0. they don't presently
1. this fixes _this_ repo as _this_ repo is presently set up
2. it's a much bigger lift to do the work to transition it to eslint
3. Opening a can of worms...

Why move from something that has just 2 dependencies to something that has hundreds of dependencies? There are just too many things that go wrong. Too many versions to track. Slower CI/CD times.

If someone _else_ wants to bring in the complexity and then make all the corresponding updates, then let them do that in another PR to transition the repository. But for now, let's just fix what's here.

I don't use eslint because I value both security and ease of use. `eslint` is an absolute monster. I won't install it on my computer because I'm morally opposed to it - it encourages non-standard features, supply chain security attacks, and overall bloat.